### PR TITLE
Added passwords for test seed students

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: db/seeds/test_seed.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-jane      = FactoryGirl.create(:user, username: "jane")
-john      = FactoryGirl.create(:user, username: "john")
-alice     = FactoryGirl.create(:user, username: "alice")
-bob       = FactoryGirl.create(:user, username: "bob")
-carol     = FactoryGirl.create(:user, username: "carol")
+jane      = FactoryGirl.create(:user, username: "jane", password: "foo")
+john      = FactoryGirl.create(:user, username: "john", password: "foo")
+alice     = FactoryGirl.create(:user, username: "alice", password: "foo")
+bob       = FactoryGirl.create(:user, username: "bob", password: "foo")
+carol     = FactoryGirl.create(:user, username: "carol", password: "foo")
```
Comment (adambray) [inline db/seeds/test_seed.rb]: every time a column is aligned a duck gets its wings.

**Comment by adambray** [comment]
:+1:

File: db/seeds/test_seed.rb (hunk 1 above)
Comment (RobertAKARobin) [inline db/seeds/test_seed.rb]: I don't think there's a scientific basis for that.

**Comment by adambray** [comment]
Thanks boo!

**Comment by andrewsunglaekim** [comment]
:+1:

**Pull request merged**